level3.md

```diff
@@ -0,0 +1,23 @@
+##  그땐 그랬지
+
+팀 프로젝트가 정말 싫었다. 혼자서 가면 더 빨리 갈 수 있는 길을 에둘러 돌아가는 느낌이었다. 기계처럼 역할을 나누고, 일정을 나누는 것. 팀 내에서 자신이 맡은 역할만을 소화하는 것. 마지막으로 결과물을 취합하는 것. 이 모든 과정이 뚝뚝 끊기는 느낌이었다. 결과물을 취합하기 전까지는 아무것도 할 수 없었다. 그럴 때마다 혼자서도 할 수 있는 일을 굳이 왜 나눠서 해야 할까 답답함을 느끼기도 했다. 우아한테크코스에 오기 전 경험했던 대다수의 팀 프로젝트가 그러했다. 
+
+갈등이 정말 두려웠다. 팀 프로젝트에서 갈등은 반드시 피해야 할 존재라고만 생각했다. 갈등은 팀을 와해시키고, 서로의 감정을 상하게 하는 요소라고 생각했기 때문이다. 갈등이 발생한 뒤에 해결하기보다는 애초에 갈등을 막는 것만이 가장 좋은 방법이라 생각했다. 갈등의 씨앗이 보이기만 하면 어떻게든 짓누르려 했고, 갈등이 발생하는 팀 프로젝트는 실패라고 생각했다. 
+
+## 나보다는 우리
+
+장기간 팀 프로젝트를 하는 경험은 이번이 난생처음이었다. 아침 데일리 미팅을 시작으로 저녁 데일리 회고, 이후 추가적인 개발까지 하루의 대부분을 팀원들과 함께했다. 심지어 점심, 저녁까지도 팀원들과 함께 먹었다. 기계처럼 역할과 일정을 나눴던 이전의 팀 프로젝트와 다르게 팀원들과 대면하는 시간이 많아졌다. 이에 따라 갈등의 발생 빈도 역시 자연스레 높아졌다. 하지만 그동안 생각했던 갈등의 이미지와는 정반대였다. 공동의 목표를 달성하기 위해 어떤 방안이 더 나을지 고민하는 과정에서 나오는 사소한 의견 충돌이 대부분이었고, 금세 해소되었다. 또한 이 과정에서 다채로운 의견들이 쏟아졌고 의견을 절충하는 과정에서 서비스가 좀 더 나은 방향으로 발전할 수 있었다. 그동안 왜 갈등을 무조건 금기시하고 피했을까? 후회가 드는 순간이었다.
+
+레벨 3에 들어오면서 유독 많이 사용했던 단어가 있다. 누구 한 명뿐만이 아니라 팀원 모두가 그랬다. 바로 '우리'라는 단어다. '우리' 이렇게 바꿔보는 건 어떨까? '우리' 서비스에서 이게 굳이 필요할까? '우리' 우선순위를 생각해보자. 레벨 2 방학식 날 서로 어색한 모습으로 모여 자신의 닉네임을 소개했던 팀원들은 이제 하나의 팀으로서 우리라는 단어를 자연스레 사용하고 있었다. 부르는 호칭만 우리가 된 것은 아니었다. 백엔드 크루가 프론트엔드 영역 개발에 참여하기도 했고, 프론트엔드 크루가 백엔드 영역 개발에 참여하기도 했다. 팀 내에서의 역할 역시도 우리가 된 것이었다. 팀의 목표를 달성하기 위해서라면 서로의 영역을 구분하지 않았다.  
+
+레벨 3의 찐한 협업을 겪으며 크게 깨달은 것이 있다. 갈등의 발생은 자연스러운 것이며, 갈등은 무조건 피해야 할 대상은 아니라는 것이다. 무엇보다도 중요한 점은 갈등을 마주했을 때 그것을 어떻게 해결하고 대처해 나갈지가 훨씬 더 중요하다는 점이다. 그동안 가지고 있었던 갈등에 대한 이미지는 팀 프로젝트에 대한 잘못된 선입견에서 비롯되었다. 기계처럼 역할을 나누려 했던 것과 갈등의 대상을 사람과 사람으로 한정 지으려 했던 잘못에서 비롯되었다. 분업보다는 협업을 지향해야 하며, 갈등의 대상은 오로지 우리의 목표여야만 한다는 점을 레벨 3에 와서야 비로소 깨닫게 되었다.
+
+## We can do it
```

Comment:
![we-can-do-7okit8](https://user-images.githubusercontent.com/78459713/184584127-e80e98a4-868b-4f34-8b17-0c7acec05727.jpeg)